### DISCUSSION TITLE
"For" option for Template Triggers

### DISCUSSION THREAD

**Discussion by lightmaster** [home-assistant/frontend#11230] [Other feature requests]
## The request

Under Automations, for Triggers: when you define a template trigger, you can click the "Edit as YAML" menu option and add a for value. This does not cause an error and functions as expected. I would propose adding a UI "for" option for Templates like there is for States.

**Comment by zsarnett** [comment]
We should not just add this as Text field though. We should make the field more descriptive / better designed for how it is used
